### PR TITLE
Use `sdk/telemetry` instead of pdata for the `autosdk` probe

### PR DESCRIPTION
**Pull request by MrAlias**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,10 @@
 FROM golang:1.23.2-bullseye AS base
 
+RUN apt-get update && apt-get install -y curl clang gcc llvm make libbpf-dev
+
 WORKDIR /app
 
-RUN apt-get update && apt-get install -y curl clang gcc llvm make libbpf-dev
+COPY sdk/ /app/sdk/
```
Comment (RonFed) [inline Dockerfile:7]: Why is this needed?

**Review by RonFed — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (MrAlias) [reply Dockerfile:7]: Yes, correct.
